tests/node/index.js

```diff
@@ -0,0 +1,32 @@
+var Fs = require('fs');
+var Path = require('path');
+var Env = require('../..').Env;
+var Fetch = require('../..').fetch;
+
+
+var env = new Env('en-US', Fetch);
+var ctx = env.createContext([
+  Path.join(__dirname + '/fixtures/{locale}/entities.l20n')
+]);
+var langs = [
+  {code: 'es-ES'},
+  {code: 'en-US'},
+  {code: 'fr'}
+];
+
+ctx.fetch(langs).then(init);
+
+function init () {
+  ctx.resolve(langs, 'whatCanIAsk').then(function (value) {
+    console.log('SINGLE VALUE: ', value);
+  });
```

Comment:
I'd prefer we didn't have to pass `langs` in each time we wanted to resolve an entity. We've already defined a default locale when creating the environment and even after that we fetch the `langs` via the `context`. I would rather the `resolve` method on `context` not have the first `langs` argument or let it be optional.

Maybe we could assign the `langs` to the context once and not need to pass them in every time.